README.md

```diff
@@ -565,6 +565,59 @@ By default, rows are sorted ascendingly. Set `descending: True` to reverse this
 </details>
 
 
+<details>
+<summary><code>flatten</code></summary>
+
+Split values in a column and create a copy of the row for each value.
+```yaml
+      - operation: flatten
+        flatten_column: my_column
+        left_wrapper: '["' # characters to trim from the left of values in `flatten_column`
+        right_wrapper: '"]' # characters to trim from the right of values in `flatten_column`
+        separator: ","  # the string by which to split values in `flatten_column`
```

Comment:
I wonder if  we want to allow regex to be passed for any of these arguments.

`pandas.Series.str.split()` has an optional boolean `regex` argument that could handle this logic.

Reply:
as we discussed, I think we can skip regex processing (for now, at least)